migrations/188_workspaces__port__add.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE workspaces ADD COLUMN IF NOT EXISTS port smallint;
```

Comment:
`smallint` is signed, so storing ports above 32K is going to be weird. I suggest just using `integer`.

Reply:
fixed in defff526a67fe0ef819f92991817ee3335b187e9